lib/pengine/unpack.c

```diff
@@ -550,8 +550,7 @@ expand_remote_rsc_meta(xmlNode *xml_obj, xmlNode *parent, pcmk_scheduler_t *data
     for (attr_set = pcmk__xe_first_child(xml_obj); attr_set != NULL;
          attr_set = pcmk__xe_next(attr_set)) {
 
-        if (!pcmk__str_eq((const char *) attr_set->name,
-                          PCMK_XE_META_ATTRIBUTES, pcmk__str_casei)) {
```

Comment:
This drops `pcmk__str_casei` but probably fine for an element name at least

Reply:
yep that's fine for element/attribute names (though not values, usually)